src/hooks/useForm.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+export default function useForm(initialValues){
```

Comment:
I would add a JSDoc string at the top of the hook to let other engineers understand what the goal of it is, what things it takes, and what it returns.